DEVELOP.md

```diff
@@ -7,6 +7,36 @@ Then install the dependencies and a development instance using:
 ```bash
 make build
 ```
+
+### Start Local Server
+
+Start Plone, on port 8080, with the command:
+
+```bash
+make start
+```
+
+#### Keycloak
+
+The `pas.plugins.oidc` repository has a working setup for a `Keycloak` development server using `Docker` and `Docker Compose`. To use it, in a terminal, run the command:
+
+```bash
+make keycloak-start
+```
+
+There are two realms configured `plone` and `plone-test`. The later is used in automated tests, while the former should be used for your development environment.
+
```

Comment:
Add that Keycloak starts on http://127.0.0.1:8180 in case some wants to enter and see what's there.

There is also documentation on README.md about how to run a Keycloak server. It should be updated to let the user they can test with this included Dockerfiles.

Add that the "plone" realm's client id is `plone` and client secret `12345678`

Add that the `openid` scope needs to be configured in order to use the realm

Reply:
I will add this info there.

Reply:
It is not the ideal README, but it seems to be better now :-)